src/test/java/org/terracotta/statistics/derived/histogram/BarSplittingBiasedHistogramPreciseTest.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Shouldn't we test on threaded value insertion somewhere?

Reply:
Threaded?  The only thing is going to need to be locked outside of this code.

Reply:
Oh! Please specify it somewhere in the javadoc. We really should start using @NotTheadSafe